packages/tokens/src/global/font-sizes.ts

```diff
@@ -0,0 +1,27 @@
+export interface FontSizesProps extends Array<number> {
+  xxsmall?: number;
+  xsmall?: number;
+  small?: number;
+  medium?: number;
+  large?: number;
+  xlarge?: number;
+  xxlarge?: number;
+  xxxlarge?: number;
+  huge?: number;
+}
+
+const fontSizes: FontSizesProps  = [10, 12, 14, 16, 20, 24, 32, 40, 48];
+
+[
+  fontSizes.xxsmall,
+  fontSizes.xsmall,
+  fontSizes.small,
+  fontSizes.medium,
+  fontSizes.large,
+  fontSizes.xlarge,
+  fontSizes.xxlarge,
+  fontSizes.xxxlarge,
+  fontSizes.huge,
+] = fontSizes;
+
+export default fontSizes;
```

Comment:
Why not?
```suggestion
const fontSizes = {
  xxsmall: 10,
  xsmall: 12,
  small: 14,
  medium: 16,
  large: 20,
  xlarge: 24,
  xxlarge: 32,
  xxxlarge: 40,
  huge: 48,
};

export type FontSizesProps = typeof fontSizes;
```

The same suggestion for other approaches.

Reply:
In addition to @MicaelRodrigues suggestion, I would add the `as const` at the end of the `fontSizes` object, like this:


```ts
const fontSizes = {
	  xxsmall: 10,
	  xsmall: 12,
	  small: 14,
	  medium: 16,
	  large: 20,
	  xlarge: 24,
	  xxlarge: 32,
	  xxxlarge: 40,
	  huge: 48,
	} as const;
```